it/test/base/BaseISpec.scala

```diff
@@ -58,6 +61,9 @@ class BaseISpec
     post("/auth/authorise") willReturn okJson(Json.obj("nino" -> "QW123456A").toString)
   )
 
+  protected lazy val EXPECTED_LOG_MESSAGE_PATTERN: Regex =
+    raw"^\[Error] - \[([^]]+)] - \[([^:]+): (.+)]$$".r
```

Comment:
the second part of the logging, would contain the entire url.
I was wondering to narrow this to either link|balance|payments

is this possible?